src/main/cpp/t06_different.cpp

```diff
@@ -19,5 +19,15 @@
 using namespace std;
 
 int t06_different() {
-
+    int n,counter;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'counter' is not assigned a value.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4493301)